tests/test_maths/test_trapezoidal_rule.py

```diff
@@ -0,0 +1,10 @@
+from maths import trapezoidal_rule
+
+
+def test_trapezoidal_rule():
+    a = 0.0
+    b = 1.0
+    steps = 10.0
+    boundary = [a, b]
+    y = trapezoidal_rule.method_1(boundary, steps)
+    assert y == 0.3349999999999999
```

Comment:
Use `pytest.approx` as in `test_simpson_rule`, but specify the allowed absolute error:
```suggestion
    assert y == pytest.approx(1.0 / 3, abs=1e-2)
```